x-pack/test/cases_api_integration/security_and_spaces/tests/trial/internal/delete_file_attachments.ts

```diff
@@ -0,0 +1,136 @@
+/*
```

Comment:
The deletion sub privilege is only available under a trial license so these tests are specifically for a user with the deletion sub privilege.